sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetOptions.scala

```diff
@@ -94,8 +94,9 @@ object ParquetOptions extends DataSourceOptions {
     "snappy" -> CompressionCodecName.SNAPPY,
     "gzip" -> CompressionCodecName.GZIP,
     "lzo" -> CompressionCodecName.LZO,
-    "lz4" -> CompressionCodecName.LZ4,
```

Comment:
May I ask why we need to move this line?

Reply:
In order to keep the order consistent:
https://github.com/apache/spark/blob/4e78ff22d9cdcc8936a52ac98b594055123c6244/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L986-L987

Reply:
Got it. It makes sense.